modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java

```diff
@@ -12,6 +12,7 @@
 import io.swagger.v3.oas.models.parameters.Parameter;
 import io.swagger.v3.oas.models.parameters.RequestBody;
 
+import io.swagger.v3.parser.OpenAPIV3Parser;
```

Comment:
what is this change for? possibly revert changes for this file

Reply:
I'll remove